src/components/legend/index.js

```diff
@@ -12,6 +12,8 @@
 var Plotly = require('../../plotly');
 var d3 = require('d3');
 
+var Pie = require('../../traces/pie');
```

Comment:
Let's require in `pie/style_one.js` directly, so that once we split out `Pie` from the core, legend b